original.name="Continue_Invalid_2"
======
>>> main.whiley
function f(int x) -> int:
    while x > 0:
        continue
        x = x + 1
    return 0

---
E504 main.whiley 4,8:16
